app/workers/repository_maintenance_stat_worker.rb

```diff
@@ -0,0 +1,25 @@
+class RepositoryMaintenanceStatWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :repo_maintenance_stat, lock: :until_and_while_executing, unique_across_queues: true, retry: 3, on_conflict: :log
+
+    def perform(repo_id)
+        GatherRepositoryMaintenanceStats.gather_stats(Repository.find_by_id(repo_id))
+    end
+
+    def self.queue(repo_id, priority: :medium)
```

Comment:
Maybe `self.enqueue`? I think of `queue` as the thing and `enqueue` as putting an object into the queue